framework/sysdeps/unix/child_debug.cpp

```diff
@@ -866,6 +828,7 @@ static bool print_signal_info(const CrashContext::Fixed &ctx)
     if (ctx.rip != ctx.crash_address)
         message += stdprintf(", CR2 = %p", ctx.crash_address);
     if (ctx.trap_nr >= 0) {
+#ifdef __x86_64__
```

Comment:
This looks weird to me, shouldn't be better to include the if inside of the "#ifdef __x86_64__"?